frontend/js/ci.js

```diff
@@ -343,7 +343,7 @@ const displayCITable = (measurements, url_params) => {
                             <td class="td-index">${run_link_node}</td>\
                             <td class="td-index">${escapeString(label)}</td>\
                             <td class="td-index"><span title="${escapeString(created_at)}">${dateToYMD(new Date(created_at))}</span></td>\
-                            <td class="td-index">${escapeString(value)}</td>\
+                            <td class="td-index">${formatLongValue(value)}</td>\
```

Comment:
Why no escaping anymore? The custom function does nothing if a malicious string is passed in terms of replacement.

Reply:
added sanitation